brave-opentracing/README.md

```diff
@@ -0,0 +1,10 @@
+# OpenTracing API for Java
+
+This library is a Java Bridge for the API to OpenTracing.
+
```

Comment:
I'd put in a sub-bullet that the status of this is experimental. Sadly we don't have an "experimental" or "beta" annotation in brave at the moment.